### ISSUE TITLE
Downstream ASV: training terminated

### ISSUE THREAD

**Issue by YangHao97** [s3prl/s3prl#283]
Hi, I'm using wav2vec 2.0 Large model to train ASV task, but the training process always stops at the same time without error
![image](https://user-images.githubusercontent.com/71253402/150928311-c6eafe40-a50f-4a5e-aabe-5ad94748880f.png)
![image](https://user-images.githubusercontent.com/71253402/150928352-de796ccb-1143-4366-8268-7e8d58255a0b.png)

**Comment by leo19941227** [comment]
This is weird... it hangs but not crashed right?

**Comment by YangHao97** [comment]
> 

Right, it does not crashed.  Everything works fine when I train other downstream tasks, but this one is weird, it seems to be frozen.

**Comment by TCL606** [comment]
Hello, have you solved this? I have encountered the same problem when using fbank on ASV. 
```python3 run_downstream.py -n ASVfb -m train -u fbank -d sv_voxceleb1```
![a979dc9142df69ff51256ae59fe089d](https://user-images.githubusercontent.com/84725343/158520586-40adaa78-d1d1-423f-b1dc-1583596ee299.png)

**Comment by YangHao97** [comment]
> Hello, have you solved this? I have encountered the same problem when using fbank on ASV. `python3 run_downstream.py -n ASVfb -m train -u fbank -d sv_voxceleb1` ![a979dc9142df69ff51256ae59fe089d](https://user-images.githubusercontent.com/84725343/158520586-40adaa78-d1d1-423f-b1dc-1583596ee299.png)

I have not solved this issue, it still exists.

**Comment by andi611** [comment]
I think this is a machine-dependent issue and not a general issue.
We have never encountered this.

**Comment by sausage-333** [comment]
I don't know why, but I've encountered this issue when I use 3080Ti GPU.
